tools/notification-configuration/ci.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/notification-configuration
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/notification-configuration
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
+    parameters:
+      ToolDirectory: notification-configuration
```

Comment:
I assume this will cause all the projects in that directory to build I'm good with that but wanted to call it out incase that wasn't what you expected.